modules/thermal_hydraulics/joss_paper.md

```diff
@@ -77,7 +112,7 @@ may declare new control data that is not associated with input parameters and
 may retrieve control data declared in other control logic objects, allowing
 control operations to be chained together, which is not possible in the standard
 `Controls` system. This is necessary to mirror real control systems in thermal
-hydraulic systems, which may have many controls in series.
+hydraulic systems, which may have complex networks of controllers.
```

Comment:
TODO: Need to mention that loops are not allowed, so network may not be the right word here. DAGs (directed acyclic graphs) are allowed.